tests/fields/test_basic.py

```diff
@@ -111,6 +108,3 @@ class Query:
     assert instance.the_list == []
     fields = Query._type_definition.fields
     assert [field.default_value for field in fields] == [3, []]
-
-    with pytest.raises(InvalidDefaultFactoryError):
-        strawberry.field(default_factory=round)
```

Comment:
why is this not raising anymore?

Reply:
I thought that this is redundant since it is pure dataclasses logic and has nothing to do with strawberry am I wrong?

Reply:
that's a good point! I'd keep it though, mostly so that we don't accidentally remove this dataclasses logic 😊

Reply:
This can make it easier to generate docs later on though..